build/initialize-pipeline copy.ps1

```diff
@@ -0,0 +1,19 @@
+# Get major, minor and patchVersions
+[xml]$XMLContents = [xml](Get-Content -Path ".\common.props")
```

Comment:
Can you clarify the intent for this file? Did you intend to have the copy? We should give this a better name.

Reply:
Didn't mean to include this. Will remove it.